tests/unit/cli/inspect/test_code_list_inspector.py

```diff
@@ -100,6 +102,69 @@ def test_get_table_identifiers_for_concept_scheme_error():
     ) in str(exception.value)
 
 
+def test_get_map_code_list_uri_to_label():
+    """
+    This test checks that the input code list's URI and labels are successfully returned
+    and match the expected results without errors.
+    """
+    path_to_cube = (
+        _test_case_base_dir
+        / "code-list-with-uri-identifier-as-identifier/category.csv-metadata.json"
+    )
+    code_list_inspector = get_code_list_inspector(path_to_cube)
+
+    concept_scheme_uri = "category.csv#code-list"
+
+    result = code_list_inspector.get_map_code_list_uri_to_label(concept_scheme_uri)
+    assert result == {
+        "boosting-productivity-pay-jobs-and-living-standards": "Boosting productivity, pay, jobs and living standards by growing the private sector",
+        "spreading-opportunity-and-improving-public-services": "Spreading opportunity and improving public services",
+        "restoring-a-sense-of-community-local-pride-and-belonging": "Restoring a sense of community, local pride and belonging",
+    }
+
+
+def test_get_map_code_list_uri_to_label_duplicate_label():
+    """
+    This test checks that the function to dereference a code list's URIs to labels correctly
+    returns an error when there are duplicate labels in the input data set's `Label` column.
+    """
+    path_to_cube = (
+        _test_case_base_dir
+        / "code-list-with-uri-identifier-as-identifier/category-duplicate-labels.csv-metadata.json"
+    )
+    code_list_inspector = get_code_list_inspector(path_to_cube)
+
+    concept_scheme_uri = "category-duplicate-labels.csv#code-list"
+
+    with pytest.raises(ValueError) as exception:
+        _ = code_list_inspector.get_map_code_list_uri_to_label(concept_scheme_uri)
+
+    assert (
+        "Duplicate labels 'Restoring a sense of community, local pride and be…' in `Label` column for category-duplicate-labels.csv"
+    ) in str(exception.value)
+
+
+def test_get_map_code_list_uri_to_label_duplicate_uri():
+    """
+    This test checks that the function to dereference a code list's URIs to labels correctly
+    returns an error when there are duplicate URIs in the input data set's `Uri Identifier` column.
+    """
+    path_to_cube = (
+        _test_case_base_dir
+        / "code-list-with-uri-identifier-as-identifier/category-duplicate-uris.csv-metadata.json"
```

Comment:
This may not work on Windows. You should always do the `/` outside of the string so pathlib can use the correct delimiter for the file system. Can you do the same on the other tests?